### PR TITLE
Pass tags to inventory component

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

**Comment by karelhala** [comment]
ping @johnsonm325

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
.

File: src/App.js [hunk 1]
```diff
@@ -1,85 +1,79 @@
...
-                    <NotificationsPortal />
-                    <Routes childProps={ this.props } />
-                </PermissionContext.Provider>
-                : null
-        );
-    }
-}
+        insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+            const tags = insights.chrome?.mapGlobalFilter?.(data).filter(item => !item.includes('workloads')) || undefined;
+            dispatch(actions.setGlobalGilterTags(tags));
```
Comment (johnsonm325) [inline src/App.js:58]: Should this be `setGlobalFilterTags`?

File: src/SmartComponents/modules/actions.js [hunk 2]
```diff
@@ -101,6 +101,13 @@ function updateReferenceId(id) {
     };
 }
 
+function setGlobalGilterTags(tags = []) {
```
Comment (johnsonm325) [inline src/SmartComponents/modules/actions.js:104]: `setGlobalFilterTags`?

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
@karelhala I had to do some research on hooks to make sure I understood what was happening. I'll need help testing this. I'm not sure what has to happen to get `GLOBAL_FILTER_UPDATE` to fire off.

File: src/App.js (hunk 1 above)
Comment (karelhala) [reply src/App.js:58]: Yup, good catch!

**Comment by karelhala** [comment]
@johnsonm325 sure thing! This is to enable the global filter feature, basically allowing users to set (currently) tags that they want the UI to be prefiltered by for all applications. If you want to see the global filter on local development you have to fire `insights.chrome.enable.globalFilter()` in console and refresh page. Currently the global filter is allowed only on beta environments and this will set localStorage key to enable it ignoring beta environment. After that you can select any tag and it will prefilter the systems modal for comparision. Basically this ticket https://issues.redhat.com/browse/RHCLOUD-9580

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**